tutorials/schedule-dataflow-jobs-with-cloud-scheduler/schedule-dataflow-jobs-with-cloud-scheduler.md

```diff
@@ -0,0 +1,73 @@
+# Schedule Your Dataflow Batch Jobs With Cloud Scheduler
+
+[Cloud Dataflow](https://cloud.google.com/dataflow) is a managed service for handling 
+both streaming and batch jobs. For your streaming jobs, you just need to launch them once without worrying about operating them afterwards. 
+However, for your batch jobs, you probably need to trigger them based on certain conditions.
+
+In the post, I will show you how you can leverage [cloud scheduler](https://cloud.google.com/scheduler/) to schedule your Dataflow batch jobs. You can find all the code in this [repo](https://github.com/zhongchen/GCP-Demo/tree/master/demos/scheduler-dataflow-demo).
+
+First things first, to be able to run your Dataflow jobs on a regular basis, you need to build your Dataflow templates. 
```

Comment:
Can you add a sample of a trivial Dataflow template, so this can be a self-contained example?

Reply:
Yes. I have cloudbuild, terraform and dataflow code in the external repo. I am going to move them to this folder so that everything will be self contained.